packages/ndla-licenses/src/licenses.js

```diff
@@ -321,6 +325,37 @@ const copy = {
   abbreviation: COPYRIGHTED.toUpperCase(),
 };
 
+const na = {
+  nn: {
+    short: naNN,
+    title: 'NA - ikkje relevant',
+    userFriendlyTitle: '',
+    url: '',
+    linkText: 'NA',
+    description: 'Lisensen er meint for innhald som ikkje treng lisens.',
+  },
+  nb: {
+    short: naNB,
+    title: 'NA - ikke relevant',
+    userFriendlyTitle: '',
+    url: '',
+    linkText: 'NA',
+    description: 'Lisensen er ment for innhold som ikke trenger lisens.',
+  },
+
+  en: {
+    short: naEN,
+    title: 'NA - not applicable',
+    userFriendlyTitle: '',
+    url: '',
+    linkText: 'NA',
+    description:
+      'The license is intended for content that does not require a license.',
+  },
+  rights: [NA],
+  abbreviation: NA.toUpperCase(),
+};
+
 function licenseByLocale(license, locale) {
```

Comment:
Litt under så er det en funksjon som heter `getLicenseByNBTitle(title, locale)` klarer ikke helt å finne om den brukes lenger, ser at den ikke er helt oppdatert heller for det mangler `cc` lisenser, men usikker på om jeg skal legge til `na` der eller ikke?

Reply:
Den brukes i ndla-video-search så den må nok være med.